app/src/main/java/com/kiwi/kiwitalk/ui/home/ChatListFragment.kt

```diff
@@ -66,10 +66,9 @@ class ChatListFragment : Fragment() {
             } else {
                 binding.tvChatListEmpty.visibility = View.INVISIBLE
                 binding.rvChatList.visibility = View.VISIBLE
-                adapter.submitList(it.channels)
+                adapter.submitList(mutableListOf<Channel>().apply { addAll(it.channels) })
```

Comment:
이게 어떻게 효과가 있는건가요?

Reply:
https://bb-library.tistory.com/257
이 글을 참고해서 해결했는데 내용은 라이브러리에서 한 리스트를 변경하면서 사용할 때 리스트의 주소가 변경되지 않으면 submitList를 수행하지 않는다고 합니다.
submitList 내부입니다.
```kotlin
if (newList == mList) {
    // nothing to do (Note - still had to inc generation, since may have ongoing work)
    if (commitCallback != null) {
        commitCallback.run();
    }
    return;
}
```
그런데 toMutableList나 ArrayList(it.channels) 같은 리스트를 복사해서 새로운 리스트를 만드는 건 안 되고, 새로운 리스트에 모든 항목을 추가해서 submitList를 호출할 때만 됐습니다.
원인은 찾지 못했습니다...